frontend/src/components/Mapbox.js

```diff
@@ -58,35 +58,19 @@ const Mapbox = props => {
         normalVotingPopup.on('close',(e)=>{
           setSchedule(false)
         })
-        const evMarker = new mapboxgl.Marker()
-          .setLngLat([
-            props.voterData.ev_longitude,
-            props.voterData.ev_latitude,
-          ])
-          .setPopup(earlyVotingPopup)
-          .addTo(map)
-          .togglePopup()
-        const normalMarker = new mapboxgl.Marker()
-          .setLngLat([props.voterData.longitude, props.voterData.latitude])
-          .setPopup(normalVotingPopup)
-          .addTo(map)
-          map.flyTo({
-            center:[props.voterData.ev_longitude,props.voterData.ev_latitude],zoom:13,
-            essential:true
-          })
```

Comment:
Looks like you're removing the map markers for the early and normal voting sites. I believe these are required as we have the popups attached to them. Not sure if the popups will be able to appear without the markers. The var names aren't in use though so if you're just trying to clean up un-used variables, maybe just remove the var names and leave the marker

Reply:
Good catch, yeah I'll put this code back tonight, thanks.